t/sanity.t

```diff
@@ -564,3 +564,233 @@ upstream 127.0.0.1:1130:
 --- no_error_log
 [error]
 
```

Comment:
3 blank lines required here. We usually use the `reindex` tool from the `openresty-old` repos.